src/Image/ImageModifier.php

```diff
@@ -179,4 +179,24 @@ public function applyFilter(InterventionImage $image) {
 
         return $img->encode($encodeType, $this->quality)->__toString();
     }
+
+    public function addFromProductionWatermark($bytes)
```

Comment:
should be named "fallback" and not production for consistency

Reply:
done.